integration/securitypolicy_test.go

```diff
@@ -397,3 +397,129 @@ func TestSecurityPolicy(t *testing.T) {
 
 	testenv.Test(t, securityPolicyFeatures)
 }
+
+func TestSecurityPolicyForGraphQL(t *testing.T) {
+	eval := is.New(t)
+	const (
+		queryName     = "Query"
+		accountsField = "accounts"
+		allField      = "*"
+		fieldName     = "getMovers"
+		limit         = int64(2)
+	)
+
+	var reqCtx context.Context
+
+	securityPolicyForGraphQL := features.New("GraphQL specific Security Policy configurations").
+		Setup(func(ctx context.Context, t *testing.T, c *envconf.Config) context.Context {
+			opConfSecret := v1.Secret{}
+			err := c.Client().Resources(operatorNamespace).Get(ctx, operatorSecret, operatorNamespace, &opConfSecret)
+			eval.NoErr(err)
+
+			// Obtain Environment configuration to be able to connect Tyk.
+			tykEnv, err := generateEnvConfig(&opConfSecret)
+			eval.NoErr(err)
+
+			if tykEnv.Mode == "ce" {
```

Comment:
You might have to change this if OSS PR is merged first.

Reply:
Yes, you are right. Let's keep this review unresolved until we decide to merge one of the PRs.

Reply:
It looks good to me otherwise.